tests/ballerina-examples-test/src/main/resources/run.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Copyright (c) 2018, WSO2 Inc. (http://wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# ----------------------------------------------------------------------------
+# Run Ballerina examples.
+# ----------------------------------------------------------------------------
+
+skipTest=""
+
+# Change directory to provided path
+cd ${1}
+
+# Change skip tests based on argument
+if [ ${2} ]
+then
+    skipTest="--skiptests"
+fi
+
```

Comment:
Shall we add a comment explaining the exclution ?

Reply:
@shafreenAnfar Added
https://github.com/ballerina-platform/ballerina-lang/pull/11729/files#diff-743a9a7936d9693477622917e10dca23R33